x/concentrated-liquidity/incentives_test.go

```diff
@@ -3738,7 +3738,7 @@ func (s *KeeperTestSuite) TestGetAllIncentiveRecordsForUptime() {
 				curUptimeRecords, err := clKeeper.GetAllIncentiveRecordsForUptime(s.Ctx, poolId, supportedUptime)
 				s.Require().NoError(err)
 
-				retrievedRecordsByUptime = append(retrievedRecordsByUptime, curUptimeRecords...)
+				retrievedRecordsByUptime = append(retrievedRecordsByUptime, curUptimeRecords...) //nolint:makezero,staticcheck
```

Comment:
I remember asking this somewhere else, but dont remember if I ever got to the answer, what does makezero do here?